x-pack/filebeat/docs/inputs/input-netflow.asciidoc

```diff
@@ -107,6 +108,18 @@ IPFIX PEN zero are equivalent to changes to NetFlow fields.
 Overriding the names and/or types of standard fields can prevent
 mapping of ECS fields to function properly.
 
+[float]
+[[detect_sequence_reset]]
+==== `detect_sequence_reset`
+
+Flag controlling whether {beatname_uc} should monitor sequence numbers in the
+Netflow packets to detect an Exporting Process reset. When this condition is
+detected, record templates for the given exporter will be dropped. This will
+cause flow loss until the exporter provides new templates. If set to `false`,
+{beatname_uc} will ignore sequence numbers, which can cause some invalid flows
+if the exporter process is reset. This option is only applicable to Netflow V9
```

Comment:
So an invalid flow could occur if the template changes after the Exporter Process reset?

Reply:
Technically, yes. In practice, I understand the exporter will first send the new templates after a reset as it doesn't make any sense to start sending flows before sending templates for them.